packages/rancher-monitoring/rancher-windows-exporter/package.yaml

```diff
@@ -1,3 +1,5 @@
-url: local
-version: 104.0.0
+url: https://github.com/prometheus-community/helm-charts.git
+subdirectory: charts/prometheus-windows-exporter
+commit: 3f7bb0e012e81a6497643d855df584ad5fe8b87a
+version: 104.0.0-rc2
```

Comment:
should be `104.0.0-rc1` we haven't been dealing with different rc numbers (less than intuitive I know)

Reply:
Alright, changed back to `104.0.0-rc1`